media/html/visjsDebuggerPanel.html

```diff
@@ -324,6 +361,49 @@
       function openAllClusters() {
         openClusterStack(Object.keys(network.body.nodes).filter((nodeId) => nodeId.startsWith('cluster_')));
       }
+
+      function showAllNodes() {
+        let nodeIds = Object.keys(network.body.nodes).filter((nodeId) => {
```

Comment:
```suggestion
        let nodeIds = Object.keys(network?.body.nodes).filter((nodeId) => {
```
The button is already interactable before the network is created.